js/model/cardDataModel.js

```diff
@@ -60,14 +62,14 @@ function CardDataModel(holderName, cardNumber, expirationDate, ballance) {
     };
 
     this.getBallance = function () {
-        return ballance;
+        return self._ballance;
```

Comment:
i dont understend for now - this - div with value 500, not find where is this happens, for now change to self